plugins/chat/assets/javascripts/discourse/services/chat-api.js

```diff
@@ -286,6 +286,92 @@ export default class ChatApi extends Service {
     );
   }
 
+  /**
```

Comment:
I know adding all these endpoints are not ideal since they are not technically chat API controller endpoints, but I think it's better to just move all these into the one place for when we move these old endpoints into API controller ones, rather than having `ajax` scattered all over the place

Reply:
yes thats good, no problem